interfaces/devdoc/threadpool_requirements.md

```diff
@@ -168,33 +170,33 @@ MOCKABLE_FUNCTION(, void, threadpool_destroy_work_item, THANDLE(THREADPOOL), thr
 MOCKABLE_FUNCTION(, int, threadpool_schedule_work, THREADPOOL_HANDLE, threadpool, THREADPOOL_WORK_FUNCTION, work_function, void*, work_function_context);
 ```
 
-`threadpool_schedule_work` schedules a work item to be executed by the threadpool.
+`threadpool_schedule_work` schedules a work item to be executed by the `threadpool`.
 
-**NON_THREADPOOL_01_020: [** If `threadpool` is `NULL`, `threadpool_schedule_work` shall fail and return a non-zero value. **]**
+**NON_THREADPOOL_01_020: [** If `threadpool` is `NULL`, `threadpool_schedule_work` shall fail and return a `non-zero` value. **]**
 
-**NON_THREADPOOL_01_021: [** If `work_function` is `NULL`, `threadpool_schedule_work` shall fail and return a non-zero value. **]**
+**NON_THREADPOOL_01_021: [** If `work_function` is `NULL`, `threadpool_schedule_work` shall fail and return a `non-zero` value. **]**
 
 **NON_THREADPOOL_01_022: [** `work_function_context` shall be allowed to be `NULL`. **]**
 
 **NON_THREADPOOL_01_023: [** Otherwise `threadpool_schedule_work` shall queue for execution the function `work_function` and pass `context` to it when it executes. **]**
 
 Note: There are no guarantees regarding the order of execution for the `work_function` callbacks.
 
-**NON_THREADPOOL_01_024: [** If any error occurs, `threadpool_schedule_work` shall fail and return a non-zero value. **]**
+**NON_THREADPOOL_01_024: [** If any error occurs, `threadpool_schedule_work` shall fail and return a `non-zero` value. **]**
```

Comment:
>` [](http://example.com/codeflow?start=101&length=1)

Nitpick: We put backticks around parameter names, function names, enum names, and defined values.

We don't put them around this kind of non-zero value.

Reply:
I just wanted to make it consistent, if you see line 159 above, both `threadpool` and `non-zero` are backticked. The changes are not by me.

Reply:
I am reverting it. But non-zero and NULL value returns are just indicators about the return values and readable to the end user if in backticks.